s3/src/test/java/akka/stream/alpakka/s3/javadsl/S3ClientTest.java

```diff
@@ -58,7 +59,8 @@ public String getRegion() throws SdkClientException {
             Some.apply(proxy),
             credentials,
             regionProvider("us-east-1"),
-            false
+            false,
+            akka.japi.Option.<String>none().asScala() //FIXME
```

Comment:
Still needs fix to allow a java Optional?

Reply:
What about `scala.Option.empty()`?
We should add Java API to S3Settings to bridge the case class. (see e.g. JmsSourceSettings), but it doesn't need to be done in the PR.